<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QDialog" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inicio de sesión</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #03312E;
    background-image: url(fondo.png);
    background-repeat: no-repeat;
    background-position: center;
}

QLabel {
    color: #A8F9FF;
    font-size: 11pt;
}

QLabel#title_label {
    font-size: 18pt;
    font-weight: bold;
    color: #A8F9FF;
}

QLabel#subtitle_label {
    font-size: 9pt;
    color: #9AE5E6;
}

QLabel#error_label {
    color: #A8F9FF;
    font-weight: bold;
}

QLineEdit {
    background-color: rgba(3, 49, 46, 210);
    border: 1px solid #037171;
    border-radius: 6px;
    padding: 4px 8px;
    color: #A8F9FF;
    selection-background-color: #037171;
}

QPushButton {
    background-color: #037171;
    color: #121619;
    border-radius: 8px;
    padding: 6px 14px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #9AE5E6;
}

QPushButton:pressed {
    background-color: #A8F9FF;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="title_label">
     <property name="text">
      <string>PMDA</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="subtitle_label">
     <property name="text">
      <string>Plataforma Multimodal de Datos Biomédicos</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_top">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_user">
       <property name="text">
        <string>Usuario:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="username_input">
       <property name="placeholderText">
        <string>Ingresa tu usuario</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_password">
       <property name="text">
        <string>Contraseña:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="password_input">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Ingresa tu contraseña</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_confirm">
       <property name="text">
        <string>Confirmar contraseña:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="confirm_password_input">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Repite la contraseña</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_mid">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer_left">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="login_button">
       <property name="text">
        <string>Iniciar sesión</string>
       </property>
       <property name="minimumWidth">
        <number>110</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="register_button">
       <property name="text">
        <string>Registrarse</string>
       </property>
       <property name="minimumWidth">
        <number>110</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_right">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_bottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="error_label">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
